tests/services/conftest.py

```diff
@@ -16,15 +16,6 @@
 from invenio_vocabularies.records.api import Vocabulary
 
 
-@pytest.fixture()
```

Comment:
it was repeated with base `conftest.py`, and needed to be of scope `module` to work with `example_records`